aws_lambda_powertools/utilities/parameters/appconfig.py

```diff
@@ -32,23 +32,23 @@ class AppConfigProvider(BaseProvider):
     -------
     **Retrieves the latest configuration value from App Config**
 
-        >>> from aws_lambda_powertools.utilities.parameters import AppConfigProvider
-        >>> appconf_provider = parameters.AppConfigProvider(environment="my_env", application="my_app")
+        >>> from aws_lambda_powertools.utilities import parameters
+        >>> provider = parameters.AppConfigProvider(environment="my_env", application="my_app")
```

Comment:
__Nitpick__: can we keep appconf_provider here for consistency with the other providers? E.g. in dynamoDB:

```
        >>> from aws_lambda_powertools.utilities.parameters import DynamoDBProvider
        >>> ddb_provider = DynamoDBProvider("ParametersTable")
        >>>
        >>> value = ddb_provider.get("my-parameter")
        >>>
        >>> print(value)
        My parameter value
```

Reply:
Sure i can change it back